Transmission.xcodeproj/project.pbxproj

```diff
@@ -3022,15 +3038,15 @@
 			files = (
 			);
 			inputPaths = (
-				"third-party/miniupnpc/VERSION",
-				"third-party/miniupnpc/miniupnpcstrings.h.in",
+				"third-party/miniupnp/miniupnpc/VERSION",
```

Comment:
nit: If you up to improve some things here in this iteration, you can declare `third-party/miniupnp/miniupnpc/updateminiupnpcstrings.sh` as an input also, as changes in this file should invalidate build step.

Reply:
I know. I've cherry picked it from #6591 which is waiting for your code review. ;)

Reply:
Oh, sorry, added a question there.
You can resolve this now.